deployment/hab_pki/modules/hab_pki/main.tf

```
@@ -33,7 +33,7 @@ resource "google_project_service" "privateca_api" {
 # KMS key rings & data sources
 resource "google_kms_key_ring" "hab" {
   location = var.signing_keyring_location
-  name     = "hab-ci"
+  name     = format("hab-%s", var.env)
```

```suggestion
  name     = "hab-${var.env}"
```

Does this work? It's the style I used throughout the other configs, and that you're already using in the hcl files.

---

bah.

---

🐑